diff.go

```diff
@@ -487,7 +487,9 @@ const (
 
 	DiffIgnoreWhitespace       DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE
 	DiffIgnoreWhitespaceChange DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_CHANGE
-	DiffIgnoreWitespaceEol     DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
+	DiffIgnoreWhitespaceEol    DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
+	// Deprecated: use DiffIgnoreWhitespaceEol instead
+	DiffIgnoreWitespaceEol = DiffIgnoreWhitespaceEol
```

Comment:
```suggestion
	DiffIgnoreWitespaceEOL = DiffIgnoreWhitespaceEol
```

per https://github.com/golang/go/wiki/CodeReviewComments#initialisms, the convention in Go is to use uppercase for initialisms.